q2-structure/proposal-by-xdg.md

```diff
@@ -0,0 +1,278 @@
+Core Team against their will, such as for flagrant or repeated violations of a
+Code of Conduct.  Any Core Team member may send a recall request to the
+Steering Council naming the individual to be removed.  The Steering Council
+must approve or reject the recall request.  If approved, the Steering Council
+will organize a Membership Change vote to ratify the removal.
+
+If the removed member is also on the Steering Council, then they are removed
+from the Steering Council as well.
+
+### Inactivity
+
+Core Team members who have stopped contributing are encouraged to declare
+themselves "inactive". Inactive members do not nominate or vote.  Inactive
+members may declare themselves active at any time, except when a vote has been
+proposed and is not concluded.  Eligibility to nominate or vote will be
+determined by the Vote Administrator.
+
+To record and honor their contributions, inactive Core Team members will
+continue to be listed alongside active members.
+
+### No Confidence in the Steering Council
+
+The Core Team may remove either a single Steering Council member or the entire
+Steering Council via a No Confidence Vote.
+
+A No Confidence Vote is triggered when a Core Team member calls for one
+publicly on an appropriate project communication channel, and another Core
+Team member seconds the proposal.
+
+If a No Confidence Vote removes all Steering Council members, the Vote
+Administrator of the No Confidence Vote will then administer an election
+to select a new Steering Council.
+
+### Amending Perl Rules of Governance
+
+Any Core Team member may propose amending the Perl Rules of Governance by
+sending a proposal to the Steering Council.  The Steering Council must decide
+to approve or reject the proposal.  If approved, the Steering Council will
+administer an Amendment Vote.
+
+### Rules for Voting
+
+Membership Change, Amendment, and No Confidence Votes require 2/3 of
+participating votes from Core Team members to pass.
+
+A Vote Administrator must be selected following the rules in the "Vote
+Administrator" section.
+
+The vote occurs in two steps:
+
+1. The Vote Administrator describes proposal being voted upon.  The Core Team
+   then may discuss the matter in advance of voting.
+
+2. Active Core Team members vote in favor or against the proposal.  Voting is
+   performed anonymously.
+
+For a Membership Change Vote, each phase will last one week.  For Amendment and
+No Confidence Votes, each phase will last two weeks.
+
+## The Steering Council
+
+The Steering Council is a 5-person committee, elected by the Core
+Team.  Candidates are not required to be members of the Core Team.  Non-member
+candidates are added to the Core Team if elected as if by a Membership Change
+Vote.
+
+References to specific elections are explained in the "Rules for Elections" section.
+
+### Powers
+
+The Steering Council has broad authority to make decisions about the
+development of the Perl language, the interpreter, and all other components,
```

Comment:
These words scoping the council's power are awkward. I really think that saying "project" just like PEP-13 does would be significantly less problematic.

Reply:
It's the language agreed in Q1 to intentionally constraint the scope of "the project".

Reply:
We agreed on those words for Q1. That doesn't mean those details need to go here?